docs/installation.md

```diff
@@ -15,6 +16,9 @@ First, add our official ralph repository:
     sudo apt-key adv --keyserver  hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
     sudo sh -c "echo 'deb https://dl.bintray.com/vi4m/ralph wheezy main' >  /etc/apt/sources.list.d/vi4m_ralph.list"
 
+After, install the secure transport for accessing repository :
```

Comment:
I've installed Ralph from deb yesterday on clean debian and first of all I have to run `sudo apt-get update && sudo apt-get install apt-transport-https`, then add new apt-key (commands above), and at the end install ralph-core (`sudo apt-get install ralph-core`). So IMO this should be placed before adding new apt-key.